TappyReader/SerialTappy.swift

```diff
@@ -104,7 +136,7 @@ public class SerialTappy : NSObject, Tappy {
         removeUnparsablePacketListener()
     }
     
-    private func notifyListenerOfStatus(status : TappyStatus){
+    @objc private func notifyListenerOfStatus(status : TappyStatus){
```

Comment:
How come you need this annotation on a private function?

Reply:
Probably don't need this. When I was trying to get the data exposed to NativeScript I annotated pretty much everything that didn't throw an error. I didn't realize the pull request from last week was still open or I wouldn't have submitted it yet. I'll merge your code and add separate listeners before making another one